File: FoodApp/app/src/main/java/com/example/foodapp/ConnectionDB.java
package com.example.foodapp;

import java.sql.Connection;
import java.sql.DriverManager;

import android.os.StrictMode;
import android.annotation.SuppressLint;
import android.util.Log;

import java.sql.SQLException;


public class ConnectionDB {
    Connection connection;
    String ip, port, db, user, pass;
    @SuppressLint("NewApi")
    public Connection connect() {
        try {
            ip = "rombintel.go.ro";
            db = "FoodApp";
            port = "1433";
            user = "sa";
            pass = "tartar2323";
            StrictMode.ThreadPolicy policy = new StrictMode.ThreadPolicy.Builder().permitAll().build();
            StrictMode.setThreadPolicy(policy);
            String connectionURL;
            Class.forName("net.sourceforge.jtds.jdbc.Driver");
            connectionURL = "jdbc:jtds:sqlserver://" + ip + ":" + port + "/" + db + ";Connection Timeout=10";
            connection = DriverManager.getConnection(connectionURL, user, pass);
        } catch (Exception e) {
            Log.e("Error : ", e.getMessage());
        }
        return connection;}}

File: FoodApp/app/src/main/java/com/example/foodapp/ConnectionDB.kt
package com.example.foodapp

import android.annotation.SuppressLint
import android.os.StrictMode
import android.os.StrictMode.ThreadPolicy
import android.util.Log
import java.sql.Connection
import java.sql.DriverManager

class ConnectionDB {
    lateinit var connection: Connection
    var ip: String? = null
    var port: String? = null
    var db: String? = null
    var user: String? = null
    var pass: String? = null
    @SuppressLint("NewApi")
    fun connect(): Connection {
        try {
            ip = "rombintel.go.ro"
            db = "FoodApp"
            port = "1433"
            user = "sa"
            pass = "tartar2323"
            val policy = ThreadPolicy.Builder().permitAll().build()
            StrictMode.setThreadPolicy(policy)
            val connectionURL: String
            Class.forName("net.sourceforge.jtds.jdbc.Driver")
            connectionURL = "jdbc:jtds:sqlserver://$ip:$port/$db;Connection Timeout=10"
            connection = DriverManager.getConnection(connectionURL, user, pass)
        } catch (e: Exception) {
            Log.e("Error : ", e.message!!)
        }
        return connection
    }
}